pkgs/applications/editors/vim/plugins/non-generated/vectorcode-nvim/default.nix

```
@@ -0,0 +1,34 @@
+{
+  lib,
+  vimUtils,
+  vectorcode,
+  vimPlugins,
+}:
+let
+  inherit (vectorcode) src version;
+in
+vimUtils.buildVimPlugin {
+  inherit src version;
+
+  pname = "vectorcode.nvim";
+
+  sourceRoot = "${src.name}/plugin";
+
+  dependencies = [
+    vectorcode
+    vimPlugins.plenary-nvim
+  ];
```

`vectorcode` is already in `buildInputs`
```suggestion
  dependencies = [ vimPlugins.plenary-nvim ];
```

---

Fixed